regression/cbmc-java/siblingobjects2/test.java

```diff
@@ -14,7 +14,8 @@ public void main() {
   }
 
   public void next_pointers_not_null() {
-    // Not currently achievable due to recursive types
+    // Should be possible to hit (previously not achievable due to recursive
```

Comment:
Future readers won't appreciate the history, say something like "recursive type requires `java-max-input-tree-depth` >= 2` or something similar